tests/v2/validation/Jenkinsfile_support_matrix_os

```diff
@@ -0,0 +1,250 @@
+#!groovy
+node {
+    def rootPath = "/go/src/github.com/rancher/rancher/tests/v2/validation/"
+    def job_name = "${JOB_NAME}"
+    if (job_name.contains('/')) { 
+      job_names = job_name.split('/')
+      job_name = job_names[job_names.size() - 1] 
+    }
+    def testContainer = "${job_name}${env.BUILD_NUMBER}_test"
+    def imageName = "rancher-validation-${job_name}${env.BUILD_NUMBER}"
+    def testsDir = "github.com/rancher/rancher/tests/v2/validation/${env.TEST_PACKAGE}"
+    def testResultsOut = "results.xml"
+    def envFile = ".env"
+    def rancherConfig = "rancher_env.config"
+    def branch = "release/v2.7"
```

Comment:
This `branch` has to match the current and a separate file for v2.8?

Reply:
So this PR is pointing to the 2.7 branch, should it be the 2.8? Not a separate file but more like its backward port just to unify branches. This file doesn't exist on 2.7, and we don't have a backport for it, so I added this file to the PR.

Reply:
So to catch this, this PR will be stale until this: https://github.com/rancher/rancher/pull/41061 PR is merged. That way we can have the same history for both branches. After a rebase, this PR will only point to the git checkout part like the 2.8 branch.

cc: @igomez06